portal-web/test/functional/com/liferay/portalweb/tests/enduser/publications/ManagePublications.testcase

```diff
@@ -31,8 +31,6 @@ definition {
 		var portalURL = PropsUtil.get("portal.url");
 
 		task ("Enable publications and add 2 different publications") {
-			Publications.enablePublications();
```

Comment:
For the last two commits, We'd better not update our own changes with a new commit.

Reply:
Sorry about that! I thought I had cleaned these up before pushing up.